coderkevin/fresh-data
# Change endpoint path structure to resourceNames

This makes a fundamental change in the way data is stored by simplifying all endpoint structure into a flat string indexed structure (thanks @dmsnell!)

Note: This PR actually removes a net of 120 (real) lines of code and 2.1k from the bundle size. 🎉

To Test:
1. `npm install`
2. `npm run test`
3. `npm run lint`
4. `npm link`
5. cd into `examples/hello-world`, `npm install`, `npm link fresh-data`, `npm start` and verify operation.
6. cd into `examples/wp-rest-api`, `npm install`, `npm link fresh-data`, `npm start` and verify operation.
